packages/node_modules/@ciscospark/plugin-people/test/integration/spec/people.js

```diff
@@ -39,6 +39,14 @@ describe(`plugin-people`, function() {
             assert.equal(peopleResponse.emails[0], mccoy.email);
           });
       });
+
+      it(`gets the current user with "get('me')"`, () => {
```

Comment:
Maybe also add test that does `get('me')` and `get(mccoy.id)` in a promise.all. it's overkill given the current implementation, but from a black-box point of view, it'll be a good test to have in place long-term.